generator/internal/scaffold_generator.cc

```diff
@@ -259,24 +259,37 @@ package(default_visibility = ["//visibility:private"])
 
 licenses(["notice"])  # Apache 2.0
 
-SOURCE_GLOB = "**/*.cc"
-
-MOCK_SOURCE_GLOB = "mocks/*.cc"
+filegroup(
+    name = "srcs",
+    srcs = glob([
+        "*.cc",
+        "internal/*.cc",
+    ]),
+)
 
-HEADER_GLOB = "**/*.h"
+filegroup(
+    name = "hdrs",
+    srcs = glob([
+        "*.h",
+        "internal/*.h",
+    ]),
+)
 
-MOCK_HEADER_GLOB = "mocks/*.h"
+filegroup(
+    name = "mocks",
+    srcs = glob([
+        "mocks/*.h",
+    ]),
+)
 
 cc_library(
     name = "google_cloud_cpp_$library$",
-    srcs = glob(
-        include = [SOURCE_GLOB],
-        exclude = [MOCK_SOURCE_GLOB],
-    ),
-    hdrs = glob(
-        include = [HEADER_GLOB],
-        exclude = [MOCK_HEADER_GLOB],
-    ),
+    srcs = [
+        ":srcs",
+    ],
+    hdrs = [
+        ":hdrs",
+    ],
```

Comment:
If you remove the trailing comma from the one element lists, these will format down to one line ... e.g., `srcs = [":srcs"],`.

Reply:
Done.